common/src/app/common/geom/shapes/transforms.cljc

```diff
@@ -662,3 +670,17 @@
        (map (comp gpr/points->selrect :points transform-shape))
        (gpr/join-selrects)))
 
+(defn apply-group-modifiers
+  "Apply the modifiers to the group children to calculate its selection rect"
+  [group objects modif-tree]
+
+  (let [children
+        (->> (:shapes group)
+             (map (d/getf objects))
+             (map (fn [shape]
+                    (let [modifiers (get modif-tree (:id shape))
+                          shape (-> shape (merge modifiers) transform-shape)]
+                      (if (= :group (:type shape))
```

Comment:
you can't because you create an import loop app.common.pages.helpers -> app.common.geom.shapes -> app.common.geom.shapes.transforms -> app.common.pages.helpers

Reply:
LOL, we need to move this helpers somewhere for avoid this. OK for now